packages/logger/tests/unit/injectLambdaContext.test.ts

```diff
@@ -239,4 +240,339 @@ describe('Inject Lambda Context', () => {
       );
     }
   );
+
+  describe('Correlation ID', () => {
```

Comment:
Let's not go back to nested tests and keep them all flat - if they're many and we think need grouping, it's ok to create a separate file.